src/aiq/meta/pypi.md

```diff
@@ -22,30 +22,36 @@ limitations under the License.
 AgentIQ is a flexible library designed to seamlessly integrate your enterprise agents—regardless of framework—with various data sources and tools. By treating agents, tools, and agentic workflows as simple function calls, AgentIQ enables true composability: build once and reuse anywhere.
 
 ## Key Features
+AgentIQ works with any agentic framework, so you can use your current technology stack without replatforming.
 
-- [**Framework Agnostic:**](https://docs.nvidia.com/agentiq/latest/concepts/plugins.html) Works with any agentic framework, so you can use your current technology stack without replatforming.
```

Comment:
I think we should keep these key features in there unless it looks too busy. On mobile so I can't tell what it looks like

Reply:
I have added that information as text to each of the key features.